drupal/rootfs/etc/s6-overlay/scripts/install.sh

```diff
@@ -47,11 +47,11 @@ function purge_queues {
 }
 
 function configure {
-       # Work around for when the cache is in a bad state, as Drush will access
-       # the cache before rebuilding it for some dumb reason, preventing
-       # Drush from being able to clear it.
-        local params=$(/var/www/drupal/web/core/scripts/rebuild_token_calculator.sh 2>/dev/null)
-        curl -L "${DRUPAL_DRUSH_URI}/core/rebuild.php?${params}"
+ # Work around for when the cache is in a bad state, as Drush will access
+ # the cache before rebuilding it for some dumb reason, preventing
+ # Drush from being able to clear it.
+  local params=$(/var/www/drupal/web/core/scripts/rebuild_token_calculator.sh 2>/dev/null)
+  curl "${DRUPAL_DRUSH_URI}/core/rebuild.php?${params}"
```

Comment:
Not sure why removing `-L` would make anything work?

> -L, --location
              (HTTP)  If  the  server  reports  that the requested page has moved to a different location (indicated with a Location: header and a 3XX response code), this option
              makes curl redo the request on the new place. If used together with -i, --include or -I, --head, headers from all requested pages are shown.

Reply:
It runs - rebuild.php clears the cache, then redirects back to the home page.  For some reason, it's insisting on redirecting back to https://islandora.io, even though the basename is explicitly set to http://

Reply:
🤷